packages/manager/src/features/Linodes/LinodesCreate/LinodeCreateContainer.tsx

```diff
@@ -694,6 +743,52 @@ class LinodeCreateContainer extends React.PureComponent<CombinedProps, State> {
       }
     }
 
+    // Validation for VPC fields
+    if (
+      this.state.selectedVPCId !== undefined &&
+      this.state.selectedVPCId !== -1
+    ) {
+      const validVPCIPv4 = vpcsValidateIP({
+        mustBeIPMask: false,
+        shouldHaveIPMask: false,
+        value: this.state.vpcIPv4AddressOfLinode,
+      });
+
+      // Situation: Subnet not selected
+      if (
+        this.state.selectedSubnetId === undefined ||
+        this.state.selectedSubnetId === -1
+      ) {
+        return this.setState(
+          () => ({
+            errors: [
+              {
+                field: 'subnet_id',
+                reason: 'You must select a subnet',
+              },
+            ],
+          }),
+          () => scrollErrorIntoView()
+        );
+      }
+
+      // Situation: 'Auto-assign a VPC IPv4 address for this Linode in the VPC' checkbox
+      // unchecked but a valid VPC IPv4 not provided
+      if (!this.state.autoassignIPv4WithinVPCEnabled && !validVPCIPv4) {
+        return this.setState(
+          () => ({
+            errors: [
+              {
+                field: 'ipv4.vpc',
+                reason: 'Must be a valid IPv4 address, e.g. 192.0.2.0',
```

Comment:
Since we won't be able to assign an IP of 192.0.2.0 to a linode bc that wouldn't be in the RFC1918 range, maybe we should change the IP example (and change it inside the linode validation schema as well)? 
```suggestion
                reason: 'Must be a valid IPv4 address, e.g. 192.168.2.0',
```